bpf/bpf_lxc.c

```diff
@@ -460,6 +466,12 @@ static inline int handle_ipv4_from_lxc(struct __sk_buff *skb, __u32 *dstID)
 
 	reason = ret;
 
+	// Check it this is return traffic to an ingress proxy.
```

Comment:
The comment says "ingress proxy" but the code below is setting the mark for egress proxy too..?[0]

Reply:
I mixed these up when updating the comments, thanks for pointing this out :-)